features/admin/admin_journey.feature

```diff
@@ -22,20 +22,6 @@ Scenario: As an admin user who has logged in to Digital Marketplace, I wish to s
   And I click the search button for 'service_id'
   Then I am presented with the summary page for that service
 
-Scenario: As an admin user who has logged in to Digital Marketplace, I wish to search for services by DUNS number
```

Comment:
This one should be covered by the master tests - https://github.com/alphagov/digitalmarketplace-functional-tests/blob/f5372b4e008e148319ad4366521077a882a90d2c/features/admin/search.feature#L14